app/controllers/games_controller.rb

```diff
@@ -15,6 +16,18 @@ def index
     @games = Game.all
   end
 
+  def update
+    @game = Game.find_by_id(params[:id])
+    return render_not_found unless @game.present?
```

Comment:
Idea - if you change to

```
@game = Game.find(params[:id])
```
Then you won't need this line:
```
return render_not_found unless @game.present?
```

Because find will raise a `RecordNotFound` error if it can't find it. Which will render a not_found page automatically.